### PR TITLE
Added inputs to include phone info and hide the flag from the search input

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**
blackbaud/skyux-forms#191

### CODE REVIEW

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-PaulCrowder you'll  note that I did this by adding two inputs:

1. `includePhoneInfo` which adds the phone info to the returned model and adds the dial code to the result dropdown
2. `hideSelectedCountryFlag` which hides the flag from the input where the selected country shows up.

I did this as I felt like we might have circumstances in the future where we might want to hide that flag outside of the phone field and I didn't want to tie the two together. 

Happy to chat about these if needed :)

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-phone-inputs](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-phone-inputs)

**Review by Blackbaud-PaulCrowder — APPROVED** [review]

File: src/app/public/modules/country-field/country-field.component.ts [hunk 1]
```diff
@@ -135,6 +122,41 @@ export class SkyCountryFieldComponent implements ControlValueAccessor, OnDestroy
     return this._disabled;
   }
 
+  /**
+   * Indicated whether to hide the flag in the input element.
```
Comment (blackbaud-johnly) [inline src/app/public/modules/country-field/country-field.component.ts:126]: Is this supposed to be `Indicated`? Or should it be `Indicates`?

File: src/app/public/modules/country-field/country-field.component.ts [hunk 2]
```diff
@@ -135,6 +122,41 @@ export class SkyCountryFieldComponent implements ControlValueAccessor, OnDestroy
...
+   * @default "false"
```
Comment (blackbaud-johnly) [inline src/app/public/modules/country-field/country-field.component.ts:127]: Is the default value supposed to be in quote marks? I might have misunderstood @Blackbaud-SteveBrush's instructions on this, but I thought `string` properties needed single quote marks and that other types didn't need quote marks.

File: src/app/public/modules/country-field/country-field.component.ts [hunk 3]
```diff
@@ -135,6 +122,41 @@ export class SkyCountryFieldComponent implements ControlValueAccessor, OnDestroy
...
+   */
+  @Input()
+  public hideSelectedCountryFlag: boolean;
+
+  /**
+   * Indicated whether to include phone information in the selected country and country dropdown.
```
Comment (blackbaud-johnly) [inline src/app/public/modules/country-field/country-field.component.ts:133]: Is this supposed to be `Indicated`? Or should it be `Indicates`?

File: src/app/public/modules/country-field/country-field.component.ts [hunk 4]
```diff
@@ -135,6 +122,41 @@ export class SkyCountryFieldComponent implements ControlValueAccessor, OnDestroy
...
+   * @default "false"
```
Comment (blackbaud-johnly) [inline src/app/public/modules/country-field/country-field.component.ts:134]: Same question about quote marks.

File: src/app/public/modules/country-field/country-field.component.ts [hunk 5]
```diff
@@ -135,6 +122,41 @@ export class SkyCountryFieldComponent implements ControlValueAccessor, OnDestroy
...
+   */
+  @Input()
```
Comment (blackbaud-johnly) [inline src/app/public/modules/country-field/country-field.component.ts:136]: Do we need a description for this property?

File: src/app/public/modules/country-field/country-field.component.ts (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/country-field/country-field.component.ts:126]: It was a typo. Fixed.

File: src/app/public/modules/country-field/country-field.component.ts (hunk 3 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/country-field/country-field.component.ts:133]: It was a typo. Fixed.

File: src/app/public/modules/country-field/country-field.component.ts (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/country-field/country-field.component.ts:127]: I had just followed one of the other components in the library's example as I haven't done a lot of these. I removed the quotes and things seem to work; so if you think they don't need to be there then I'll leave them out :)

File: src/app/public/modules/country-field/country-field.component.ts (hunk 4 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/country-field/country-field.component.ts:134]: See above

File: src/app/public/modules/country-field/country-field.component.ts (hunk 5 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/country-field/country-field.component.ts:136]: Done. That was an existing import that just got moved around but it definitely should have had JSDocs

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-phone-inputs](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-phone-inputs)

**Pull request merged**